hail/python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -694,6 +652,15 @@
     "common_mt = mt.filter_rows(mt.variant_qc.AF[1] > 0.01)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "common_mt = common_mt.filter_rows(common_mt.variant_qc.p_value_hwe > 1e-6)"
```

Comment:
why is there both common_mt and cmt? i prefer the shorter name with explanation.

Reply:
also, why do we have gwas and linear_regression_results? I'm fine with gwas, its shorter and makes sense for the object in the context of this tutorial. I do think we should add a burden test in the future.